Tests/PointFreeTests/SubscribeTests.swift

```diff
@@ -0,0 +1,158 @@
+import Either
+import Html
+import HtmlPrettyPrint
+import HttpPipeline
+@testable import PointFree
+import PointFreeTestSupport
+import Prelude
+import Optics
+import SnapshotTesting
+import XCTest
+
+final class SubscribeTests: TestCase {
```

Comment:
I dont think we are testing the subscribe endpoint at all right now, so I added a bunch of tests